Sources/Reporting/XCTestErrorReporter.swift

```diff
@@ -0,0 +1,18 @@
+import Foundation
+import XCTest
+
+public class XCTestErrorReporter: ErrorReporter {
```

Comment:
I'm not a fan of creating classes that use the same prefix as Apple-provided classes.  Maybe rename this to something like "PactTestErrorReporter" instead.

Reply:
Or maybe these few reporting things can be named like `ErrorReporterXCTest` and `ErrorReporterNimble`.